openauto/CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
 add_library(openauto SHARED
         App.cpp
+        ../btservice/AndroidBluetoothServer.cpp
```

Comment:
can we use absolute paths instead of relative? (by using whatever var is provided)